rllib/core/rl_module/rl_module.py

```diff
@@ -111,6 +113,16 @@ def from_dict(cls, d):
             catalog_class=catalog_class,
         )
 
+    def update(self, other):
+        """Updates this spec with the given other spec. Works like dict.update()."""
+        assert type(other) is SingleAgentRLModuleSpec
+
+        self.module_class = other.module_class or self.module_class
```

Comment:
Add a comment to clarify:
"If the field is None in the other, keep the current field, otherwise update with the new value"